easypermissions/build.gradle

```diff
@@ -33,14 +33,16 @@ junitJacoco {
 }
 
 dependencies {
-    api "com.android.support:appcompat-v7:$support_library_version"
-    api "com.android.support:support-compat:$support_library_version"
-    api "com.android.support:support-fragment:$support_library_version"
+    api "androidx.appcompat:appcompat:1.0.2"
+    api "androidx.core:core:1.0.1"
+    api "androidx.fragment:fragment:1.0.0"
 
     testImplementation 'junit:junit:4.12'
     testImplementation 'com.google.truth:truth:0.42'
     testImplementation 'org.robolectric:robolectric:4.1'
-    testImplementation 'org.robolectric:shadows-supportv4:4.1'
+    // TODO
+//    testImplementation 'org.robolectric:shadows-supportv4:4.1'
```

Comment:
Why you had to remove it? Was something wrong with the SupportFragmentController after the migration?

Reply:
This library is not compatible with AndroidX and Jetifier (a lot of people online report the same problem).  So I upgraded to the new test support library and then made a small wrapper to mimic the old behavior.